wcc-contentful-app/app/controllers/wcc/contentful/app/contact_form_controller.rb

```diff
@@ -2,7 +2,11 @@
 
 class WCC::Contentful::App::ContactFormController < ApplicationController
   def create
-    address = params[:email_object_id] ? email_address(email_model) : form_model.notification_email
+    address = form_model.to_address(
```

Comment:
could this be `address = params[:email] || form_model.to_address(params[:email_object_id])` ?

Reply:
Yes, I like that better. I just changed it accordingly.